spec/support/filterable_search_behavior.rb

```diff
@@ -1,87 +1,93 @@
+# frozen_string_literal: true
```

Comment:
All changes herein are rubocop-inspired.  I was initially going to add a shared tags param test in here, but discovered the divergence in how NewsSearch set tag filters and moved the tests up into the 3 model specs - `blended_search_spec.rb`, `i14y_search_spec.rb`, and `news_search_spec.rb`.  I didn't want to back out these updates so they remain.